pkg/apis/componentconfig/v1alpha1/zz_generated.defaults.go

```diff
@@ -41,20 +43,21 @@ func RegisterDefaults(scheme *runtime.Scheme) error {
 func SetObjectDefaults_CloudControllerManagerConfiguration(in *CloudControllerManagerConfiguration) {
 	SetDefaults_CloudControllerManagerConfiguration(in)
 	SetDefaults_GenericComponentConfiguration(&in.GenericComponent)
-	SetDefaults_LeaderElectionConfiguration(&in.GenericComponent.LeaderElection)
+	configv1alpha1.SetDefaults_LeaderElectionConfiguration(&in.GenericComponent.LeaderElection)
```

Comment:
was this automatically generated or a manual fixup?

Reply:
automatically generated. As said we might want to revisit this later, but this is as-is for now

Reply:
joining to shared defaults was what @deads2k asked not to do for these common structs in https://github.com/kubernetes/community/pull/2354#issuecomment-407533863

Reply:
opened https://github.com/kubernetes/kubernetes/pull/67207 for that